pkg/sdk/poc/generator/template_executors.go

```diff
@@ -21,9 +21,18 @@ func GenerateInterface(writer io.Writer, def *Interface) {
 		if o.OptsField != nil {
 			generateOptionsStruct(writer, o)
 		}
+		if o.ResourceHelperMethods != nil {
```

Comment:
Nit: this check is not necessary - when a slice is nil, `for ... range` does not iterate.

Reply:
done